charts/kube-prometheus-stack/values.yaml

```diff
@@ -1464,6 +1464,11 @@ kubelet:
     additionalLabels: {}
     #  foo: bar
 
+    ## defines the labels which are transferred from the associated Kubernetes Service object onto the ingested metrics.
+    ## https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#servicemonitor
+    targetLabels: []
```

Comment:
What about other exporters?

Reply:
Sorry, maybe I was not clear enough.

I was talking about other exporters inside kube-prometheus-stack. There are more than just kubelet:

https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack/templates/exporters